contracts/Colony.sol

```diff
@@ -37,6 +37,12 @@ contract Colony is ColonyStorage, PatriciaTreeProofs {
     token = ERC20Extended(_token);
   }
 
+  function setTokenLocking(address _tokenLocking) public
```

Comment:
In this case external call to `ColonyNetwork` inside `claimNextRewardPayout` adds ~7000 gas, while setting the `setTokenLocking` costs 47458 gas. It will be either us spending 47k or user spending 7k more